README.md

```diff
@@ -4,6 +4,7 @@
 <a href="https://github.com/thedevdojo/wave/actions"><img src="https://github.com/thedevdojo/wave/actions/workflows/tests.yml/badge.svg" alt="Build Status"></a>
 <a href="https://github.com/thedevdojo/wave"><img src="https://img.shields.io/github/v/release/thedevdojo/wave" alt="Latest Stable Version"></a>
 <a href="https://github.com/thedevdojo/wave"><img src="https://img.shields.io/badge/license-MIT-green" alt="License"></a>
+<a href="https://herd.laravel.com/new?starter-kit=devdojo/wave"><img src="https://img.shields.io/badge/Install%20with%20Herd-f55247?logo=laravel&logoColor=white"></a>
```

Comment:
Wonder if this should be `thedevdojo/wave` to match the repo name or if you are handling this internally so it can stay `devdojo/wave`?

Reply:
The `starter-kit` value will be passed to the new Laravel Installer, which uses package names registered at packagist.

Reply:
I'm fairly certain Herd passes the starter kit parameter into the installer as the parameter for `--using` which uses `create-project` from Composer internally[^1], which expects the package name[^2] and not the repository.


[^1]: https://github.com/laravel/installer/blob/665f6f4932dcb9f0ffbe3626f77b3afed785e45d/src/NewCommand.php#L233
[^2]: https://getcomposer.org/doc/03-cli.md#create-project